frontend/containers/AuctionsFakeData.vue

```diff
@@ -75,6 +95,11 @@ export default Vue.extend({
                 }
             },
         },
+        transactionAmountDai(newAmount) {
+            if (newAmount === undefined) {
+                this.transactionAmountDai = this.selectedAuction?.totalPrice || undefined;
```

Comment:
```suggestion
                this.transactionAmountDai = this.selectedAuction?.totalPrice ?? undefined;
```

As we're working with numbers, it's better to use `??` operator to avoid automatic conversion of `0` into falsy statement. It's not very relevant here, since we're using BigNumber, but it's a good practice to avoid `||` where you want to handle only `?.` / `null` / `undefined`